backend/inspirehep/records/marshmallow/common/accelerator_experiment.py

```diff
@@ -37,10 +37,14 @@ def get_control_numbers_to_resolved_experiments_map(self, record):
         return {record["control_number"]: record for record in resolved_records}
 
     def get_resolved_record_or_experiment(self, experiment_records_map, experiment):
-        experiment_record_id = get_recid_from_ref(experiment.get("record"))
+        record_ref = experiment.get("record", None)
```

Comment:
```suggestion
        record_ref = experiment.get("record")
```

it's `None` by default :)

Reply:
I didn't change original code ;)

Reply:
I doesn't matter, could we please change it? :)

Reply:
It's changed already ;)